dbt/adapters/spark/connections.py

```diff
@@ -350,6 +350,7 @@ def open(cls, connection: Connection) -> Connection:
 
         creds = connection.credentials
         exc = None
+        handle: Any
```

Comment:
Is this truly `Any`? It looks like we're only assigning a `SessionConnectionWrapper` instance to it.

Reply:
Nope, but we are assigning it to the other `ConnectionWrapper`s too. @Fokko created #828 to create a base class for this

Reply:
The `handle` is set to other `Wrapper`s too, e.g. [`handle = PyhiveConnectionWrapper(conn)`](https://github.com/dbt-labs/dbt-spark/blob/fd831a8b9ffcbc2298dfd26609814d17a4fcec86/dbt/adapters/spark/connections.py#L381) and [`handle = PyodbcConnectionWrapper(conn)`](https://github.com/dbt-labs/dbt-spark/blob/fd831a8b9ffcbc2298dfd26609814d17a4fcec86/dbt/adapters/spark/connections.py#L457). 

I added the `Any` because [mypy](https://github.com/dbt-labs/dbt-spark/actions/runs/5484856820/job/14853429276) complained about it and we decided to solve it in #828

Reply:
Ah, yeah, if it's used with other wrappers as well, then that's fine. The alternative is a rather large type annotation that doesn't really buy us a lot.